### ISSUE TITLE
Thanks 

### ISSUE THREAD

**Issue by PauloPhagula** [tjjjwxzq/ruby-object-model-talk#1]
Seen the Youtube video on the first time you gave this talk. 

This is informative, beautiful and awesome. Thanks! 🙌

**Comment by arpitchauhan** [comment]
Agreed! Anyone interested can watch the talk [here.](https://www.youtube.com/watch?v=268UU4EpTew)

**Comment by tjjjwxzq** [comment]
Thank you! Glad you enjoyed it :smile_cat:
